src/runmode-af-xdp.c

```diff
@@ -146,6 +148,43 @@ static TmEcode ConfigSetThreads(AFXDPIfaceConfig *aconf, const char *entry_str)
     SCReturnInt(TM_ECODE_OK);
 }
 
+/**
+ * \brief function to unload an AF_XDP program
+ *
+ */
+void RunModeAFXDPRemoveProg(void)
+{
+    const char *live_dev = NULL;
+    if (live_dev == NULL)
```

Comment:
What's this check for? The variable has been defined ``NULL`` and immediately checking for ``NULL``?

Reply:
good catch. Its remnants of old code I tried. Fixing now. Will address in the next PR